src/styles/StyleUtils.ts

```diff
@@ -738,146 +621,125 @@ function extractValuesFromRGB(color) {
  */
-function getThemeBackgroundColor(bgColor = themeColors.appBG) {
+function getThemeBackgroundColor(bgColor: string = themeColors.appBG): string {
     const backdropOpacity = variables.modalFullscreenBackdropOpacity;
 
-    const [backgroundRed, backgroundGreen, backgroundBlue] = extractValuesFromRGB(bgColor) || hexadecimalToRGBArray(bgColor);
-    const [backdropRed, backdropGreen, backdropBlue] = hexadecimalToRGBArray(themeColors.modalBackdrop);
+    const [backgroundRed, backgroundGreen, backgroundBlue] = extractValuesFromRGB(bgColor) ?? hexadecimalToRGBArray(bgColor) ?? [];
+    const [backdropRed, backdropGreen, backdropBlue] = hexadecimalToRGBArray(themeColors.modalBackdrop) ?? [];
     const normalizedBackdropRGB = convertRGBToUnitValues(backdropRed, backdropGreen, backdropBlue);
     const normalizedBackgroundRGB = convertRGBToUnitValues(backgroundRed, backgroundGreen, backgroundBlue);
-    const themeRGBNormalized = convertRGBAToRGB(normalizedBackdropRGB, normalizedBackgroundRGB, backdropOpacity);
-    const themeRGB = convertUnitValuesToRGB(...themeRGBNormalized);
+    // const themeRGBNormalized = convertRGBAToRGB(normalizedBackdropRGB, normalizedBackgroundRGB, backdropOpacity);
+    // const themeRGB = convertUnitValuesToRGB(...themeRGBNormalized);
+    const [red, green, blue] = convertRGBAToRGB(normalizedBackdropRGB, normalizedBackgroundRGB, backdropOpacity);
+    const themeRGB = convertUnitValuesToRGB(red, green, blue);
 
     return `rgb(${themeRGB.join(', ')})`;
 }
 
 /**
  * Parse styleParam and return Styles array
- * @param {Object|Object[]} styleParam
- * @returns {Object[]}
  */
-function parseStyleAsArray(styleParam) {
-    return _.isArray(styleParam) ? styleParam : [styleParam];
+function parseStyleAsArray(styleParam: ViewStyle | CSSProperties | Array<ViewStyle | CSSProperties>): Array<ViewStyle | CSSProperties> {
+    return Array.isArray(styleParam) ? styleParam : [styleParam];
 }
 
 /**
  * Parse style function and return Styles object
- * @param {Object|Object[]|Function} style
- * @param {Object} state
- * @returns {Object[]}
  */
-function parseStyleFromFunction(style, state) {
-    const functionAppliedStyle = _.isFunction(style) ? style(state) : style;
+function parseStyleFromFunction(
+    style: ViewStyle | CSSProperties | ((state: PressableStateCallbackType) => ViewStyle | CSSProperties),
+    state: PressableStateCallbackType,
+): Array<ViewStyle | CSSProperties> {
+    const functionAppliedStyle = typeof style === 'function' ? style(state) : style;
     return parseStyleAsArray(functionAppliedStyle);
 }
 
 /**
  * Receives any number of object or array style objects and returns them all as an array
- * @param {Object|Object[]} allStyles
- * @return {Object[]}
  */
-function combineStyles(...allStyles) {
-    let finalStyles = [];
-    _.each(allStyles, (style) => {
+function combineStyles(...allStyles: Array<ViewStyle | CSSProperties | Array<ViewStyle | CSSProperties>>) {
```

Comment:
The way you typed it will allow using a matrix of styles, is that correct?

Reply:
Yes, it's even explained in the original jsdoc above the function